cypress/obj/loginPageObject.js

```diff
@@ -0,0 +1,59 @@
+class LoginPage{
+    visit() {
+        cy.visit('/')
+    }
+
+    getUserValidationIndicator(){
+        return cy.get('.Select.not-valid')
+    }
+
+    getLoginButton(){
+        return cy.get('[type="submit"]')
+    }
+
+    getLoginFormUserId(){
+        return cy.get('[id="loginForm.userId"]')
+    }
+
+    getUserName(userName){
+        return cy.get('[aria-label="'+ userName +'"]')
+    }
+
+    getLoginFormRoleId(){
+        return cy.get('[id="loginForm.role"]')
+    }
+
+    getUserRole(){
+        return  cy.get('[aria-label="Team Lead"]')
+    }
+
+    getSubmitButton(){
+        return cy.get('[type="submit"]')
+    }
+
+    getUrl(){
+        return cy.url().should('include', '/time-logging')
```

Comment:
function name does not maintain anything about asserting. You can do this in two ways:
1. change the function name to "shouldBeOnTimeLoggingPage()" but it is not recommended to "hide" assert in the functions, they should be in the test case. (In this case not sure if I would agree, so it is up to you)
2. move .should('include', '/time-logging') back to test case, then move '/time-logging' to a new function and return it in like that getUrl()..should('include', timeLoggingPage.getURL()). Also if you do that then there is no purpose for [return cy.url()] to stay in login page as it is cypress function and not a locator